Introduktion 2.1 (#66)
MrLongbottom/P10-Article: sections/introduction.tex

\section{Introduction}\label{sec:introduction}
% Intro to recommendation and techniques

> Broadcasting information around the world is a crucial skill, that is used every day, like news articles, television and newspapers supply people with the latest information.

Reviewer: ```suggestion
News is being broadcast every day around the world in the form of news articles, television, and newspapers, which supply people with the latest information.
```
Omskrev da jeg ikke følte at broadcasting som en 'skill' gav mening at snakke om.

> Searching and categorizing the news is becoming a bigger problem since the news is created every day, if not every hour.

Reviewer: ```suggestion
Searching and categorizing the news is becoming a bigger problem since news is created at all times.
```
Eller noget i den stil.

> As the internet, becomes more personalized, like a targeted advertisement, recommendation is seeing more use in applications, such as Netflix and Amazon.

Reviewer: ```suggestion
As the internet becomes more personalized, as seen with targeted advertisements, recommendation systems are seeing more use in applications, such as Netflix and Amazon.
```

Author: ```suggestion
As the internet becomes more personalized, as seen with targeted advertisements, recommender systems are seeing more use in applications, such as Netflix and Amazon.
```

> Within recommendation, there are two main machine learning techniques: Collaborative filtering and Content-based filtering.

Reviewer: ```suggestion
Within recommendation, there are two main machine learning techniques: collaborative filtering and content-based filtering.
```

> These two techniques are based on assumption that similar users like similar items (collaborative) or one user likes certain types of items(content-based).

Reviewer: ```suggestion
These two techniques are based on assumption that similar users like similar items (collaborative) or one user likes certain types of items (content-based).
```

> The recommendation task can be very specific depending on the dataset that is used.

Reviewer: Jeg tror ikke helt jeg forstår sætningen. Hvornår er en recommendation ikke specifik/hvad menes der med det?

Author: jeg kunne ikke lige finde på en god overgang

Reviewer: ```suggestion
```
Så tænker jeg umiddelbart bare at slette den.

> Broadcasting information around the world is a crucial skill, that is used every day, as news articles, television and newspapers supply people with the latest information.
> Searching and categorizing the news is becoming a bigger problem, since news are created every day, if not every hour.
> As the internet, becomes more personalized, like targeted advertisement, recommendation is seeing more use in applications, such as Netflix and Amazon.
> Within recommendation, there are two main machine learning techniques: Collaborative filtering and Content based filtering.
> These two techniques are based on assumption that similar users like similar items (collaborative) or one user likes certain types items(content based).
> The recommendation task can be very specific depending on the dataset that is used.
> We focus on creating a better environment for recommendation of news articles.

Author: ```suggestion
Broadcasting information around the world is a crucial skill, that is used every day, like news articles, television and newspapers supply people with the latest information.
Searching and categorizing the news is becoming a bigger problem since the news is created every day, if not every hour.
As the internet, becomes more personalized, like a targeted advertisement, recommendation is seeing more use in applications, such as Netflix and Amazon.
Within recommendation, there are two main machine learning techniques: Collaborative filtering and Content-based filtering.
These two techniques are based on assumption that similar users like similar items (collaborative) or one user likes certain types of items(content-based).
The recommendation task can be very specific depending on the dataset that is used.
We focus on creating a better environment for the recommendation of news articles.
```

> These two techniques are based on assumption that similar users like similar items (collaborative) or one user likes certain types of items (content-based).

Reviewer: ```suggestion
These two techniques are based on the assumption that similar users like similar items (collaborative) or one user likes certain types of items (content-based).
```

> We focus on creating a better environment for the recommendation of news articles.

Reviewer: ```suggestion
We focus on creating a better environment for the content-based recommendation of news articles.
```
Måske. Hvis det skal hænge lidt mere sammen med den tidligere sætning.

> One particular field, which covers a variety of different use cases and applications in the field of topic modeling.

Reviewer: ```suggestion
One particular field, which covers a variety of different use cases and applications, such as being useful for recommendations, is the field of topic modeling.
```

> Topic modeling has been used for finding topics within a collection of documents, to either annotate or categorize certain collections of texts\cite{red_article_blei}.   

Reviewer: ```suggestion
Topic modeling has been used for finding topics within a collection of documents to either annotate or categorize certain collections of texts~\cite{Probabilistic_Topic_Models}.
```

> One particular field, which covers a variety of different use cases and application is the field of topic modeling.
> Topic modeling has been used for finding topics within a collection of document, to either annotate or categories certain collections of texts\cite{red_article_blei}.   
> The \gls{lda} model is a well known topic model which generates topics and topic distributions for documents based on the words in documents~\cite{blei2003latent}, without the use of metadata.

Author: ```suggestion
One particular field, which covers a variety of different use cases and applications in the field of topic modeling.
Topic modeling has been used for finding topics within a collection of documents, to either annotate or categorize certain collections of texts\cite{red_article_blei}.   
The \gls{lda} model is a well-known topic model which generates topics and topic distributions for documents based on the words in documents~\cite{blei2003latent}, without the use of metadata.
```

> The \gls{lda} model is a well-known topic model which generates topics and topic distributions for documents based on the words in documents~\cite{blei2003latent}, without the use of metadata.

Reviewer: ```suggestion
The \gls{lda} model is an often-cited topic model which generates topics and topic distributions for documents based on the words in documents~\cite{blei2003latent}, without the use of metadata.
```
Synes de havde snakket om at well-known ikke er defineret.

Author: ```suggestion
The \gls{lda} model is a well-cited topic model which generates topics and topic distributions for documents based on the words in documents~\cite{blei2003latent}, without the use of metadata.
```

They also show that the Author-Topic combination yields better and more coherent topics, which begs the question of whether any other document-related data can be applied similarly.
However, they only test their algorithm on scientific papers, where usually the authors write about the same subject (their research field), which might not be the case for other fields, like journalism. 

% Intro to meta data

> Data is everywhere and it is used in many different scenarios and use cases. 

Reviewer: Skal muligvis have en lidt mere flydende overgang fra at snakke om topic models.

Reviewer: ```suggestion
The dataset used with a topic model can have a large impact on the topics generated.
```
Forslag
